### PR TITLE
Adding Orchard.ResponseCache module

### PR DESCRIPTION
**Pull request by sebastienros**
Even faster than static websites.

### CODE REVIEW

**Comment by agriffard** [comment]
When I enable the module 'Response Cache' and test in another browser, not being authenticated, the header is still `Cache-Control :"max-age=0"`.

If I add a breakpoint in Startup.cs, when I enable the module, I can see the process going in the `Configure `method.

**Comment by sebastienros** [comment]
Chrome always sends a `max-age:0` for `localhost`. Try Edge, or `curl`

File: src/Orchard.Cms.Web/Modules/Orchard.ResponseCache/Orchard.ResponseCache.csproj [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    <TargetFramework>netstandard1.6</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.ResponseCaching" Version="1.1.1" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\OrchardCore\Microsoft.AspNetCore.Modules.Abstractions\Microsoft.AspNetCore.Modules.Abstractions.csproj" />
+    <ProjectReference Include="..\..\..\OrchardCore\Microsoft.AspNetCore.Mvc.Modules.Abstractions\Microsoft.AspNetCore.Mvc.Modules.Abstractions.csproj" />
```
Comment (Jetski5822) [inline src/Orchard.Cms.Web/Modules/Orchard.ResponseCache/Orchard.ResponseCache.csproj]: Why do you rely on the MVC Modules abstraction?

Surely you only need the first one?

**Comment by PinpointTownes** [comment]
@sebastienros FYI, I gave this module a try and it looks like it has no effect when the request is handled by another module instead of being handled by MVC. I'd bet that's because the response caching module is invoked too late in the pipeline.

Do you have plans to introduce some sort of "ordering" in the modules, or is response caching something the modules that want to use it should handle themselves?

**Comment by sebastienros** [comment]
Yes, plans, we definitely need it.

Also I noticed it didn't work if I didn't run the site from command line (kestrel?) in release.

**Pull request merged**